### DISCUSSION TITLE
Cannot Create Codespace from Github Web Editor in PR

### DISCUSSION THREAD

**Discussion by TheTripleV** [community/community#6188] [Codespaces]
I pressed `.` on a pull request to open up the web editor. From here, In the terminal, I clicked on `Continue Working on...` -> `Create New Codespace`.

This fails with
> Create New Codespace: Request failed with status 400: Oid is missing, please ensure you specified a valid ref (branch name or commit SHA) or pull request. Correlation ID: undefined

However, from the pull request page, clicking on `< > Code` -> `New Codespace` does work.

**Comment by bamurtaugh** [comment]
@jkeech any thoughts?

**Reply by jkeech** [reply]
Thanks for reporting! We have a bug tracking this issue internally. It's a problem related to opening the PR from a fork and how the fork repo's name and ref are handled. For now, a workaround would be to create the codespace from the PR in github.com instead of from the PR in github.dev (just change .dev to .com in the url and then create the codespace).
